core/src/main/java/com/linecorp/armeria/server/logging/DefaultAccessLoggerStrategy.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server.logging;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.function.Function;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linecorp.armeria.server.VirtualHost;
+
+/**
+ * .
+ */
+public final class DefaultAccessLoggerStrategy {
```

Comment:
Could be package-local

Reply:
this class is in com.linecorp.armeria.server.logging and used in AbstractVirtualHostBuilder, ServerBuilder in com.linecrop.armeria.server. I put the class in logging because it is about logger. Do you think its package name needs to be changed? Or should I just leave it?